db/migrate/20210804184031_add_is_hidden_to_section_due_date.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsHiddenToSectionDueDate < ActiveRecord::Migration[6.1]
+  def change
+    add_column :section_due_dates, :is_hidden, :boolean
```

Comment:
I would recommend making this not nullable and false by default. This will allow you to simplify your code in `app/views/shared/_assignments_dropdown_menu.html.erb` (for example) because you can just check for boolean values and won't have to deal with `nil`.

Reply:
note to self: decided to make this nullable where null represents "default" - i.e. the is_hidden property of the associated assessment